katas/content/random_numbers/index.md

```diff
@@ -96,13 +96,14 @@ This knowledge is sufficient to implement a simple random number generator!
 
 @[section]({"id": "random_numbers__whats_next", "title": "What's Next?"})
 
-Congratulations! In this Kata you have created a truly random number generator using quantum computing. Here are a few key concepts to keep in mind:
+Congratulations! In this kata you have created a truly random number generator. Here are a few key concepts to keep in mind:
```

Comment:
```suggestion
Congratulations! In this kata you have created a random number generator. Here are a few key concepts to keep in mind:
```
(We're telling them that it's not truly random in the next line)

Reply:
No more pride moments :(